js/game.js

```diff
@@ -46,6 +45,15 @@ startgame = function () {
 
 
 	document.addEventListener('keydown', (event) => {
+		if (event.keyCode === 27) {
+			var escapeEvent = document.getElementById('close');
+			escapeEvent.click()
+		}
+
+		// if (snakeDirection === "right" && (event.keyCode === 37 && event.keyCode === 38)) {
```

Comment:
remove un wanted code.

Reply:
I have commented this code because I was implementing the two key press error. It's not working now but will update soon.